### PR TITLE
Address QIT PHPStan test errors

### LINKED ISSUE #117: Address QIT PHPStan test errors
The Extension has Woo QIT set up as a GitHub Action to check the PHPStan test on PRs. But, it is currently failing with 62 File Errors on the trunk branch. The result of the test run can be found[ here](https://href.li/?https://compatibilitydashboard.wpcomstaging.com/?qit_results=4530542.OCKcnUj6PhpGaxWVOCDOR6ayboJ1xiFoOjebfc6hJ0D2CChvAJaEyEnGp9NAOrfy). This aims to resolve these errors to ensure the test passes on the trunk branch.

### PR DESCRIPTION
**Pull request by iamdharmesh**
### All Submissions:

* [x] Does your code follow the [WooCommerce Sniffs](https://github.com/woocommerce/woocommerce-sniffs/) variant of WordPress coding standards?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?
* [ ] Will this change require new documentation or changes to existing documentation?

---

### Changes proposed in this Pull Request:
PR fixes PHPStan QIT Test errors by making needed changes.

Closes #117 

### Steps to test the changes in this Pull Request:
1. Verify that the QIT Tests GH action Pass
1. Regression testing of extension.

### CODE REVIEW

File: includes/Framework/Api/Base.php [hunk 1]
```diff
@@ -598,6 +598,27 @@ abstract protected function get_new_request( $args = array() );
 	 */
 	abstract protected function get_plugin();
 
+	/**
+	 * Set the Content-Type request header
+	 *
+	 * @since x.x.x
+	 * @param string $content_type
```
Comment (dkotter) [inline includes/Framework/Api/Base.php]: Can we add a description for this parameter?

File: includes/Framework/PaymentGateway/PaymentTokens/Payment_Gateway_Payment_Tokens_Handler.php [hunk 2]
```diff
@@ -212,7 +212,7 @@ public function create_token( \WC_Order $order, $response = null, $environment_i
 	 * @param int $user_id user identifier
 	 * @param Payment_Gateway_Payment_Token $token the token
 	 * @param string|null $environment_id optional environment id, defaults to plugin current environment
-	 * @return bool|int false if token not added, user meta ID if added
+	 * @return void
```
Comment (dkotter) [inline includes/Framework/PaymentGateway/PaymentTokens/Payment_Gateway_Payment_Tokens_Handler.php]: My preference is to just remove the `@return` statement if we aren't returning anything, though not sure if that aligns with the PHPStan standards

File: includes/Gateway/Digital_Wallet.php [hunk 3]
```diff
@@ -1072,7 +1072,7 @@ public function check_apple_pay_verification_file() {
 	 * If the store has been registered, keep verifying the registration of the current connected account and domain every hour.
 	 *
 	 * @since 2.3
-	 * @return string
+	 * @return void
```
Comment (dkotter) [inline includes/Gateway/Digital_Wallet.php]: Same note here

**Review by dkotter — COMMENTED** [review]
Few minor suggestions

File: includes/Framework/Api/Base.php (hunk 1 above)
Comment (iamdharmesh) [reply includes/Framework/Api/Base.php]: Addressed in https://github.com/woocommerce/woocommerce-square/pull/131/commits/1bc031a03f81b8272a916c21eb54a4b7a3df2e12

**Review by dkotter — APPROVED** [review]

**Comment by qasumitbagthariya** [comment]
Regression / Smoke Test Report  ✅ 
-------

Tested with Archive File created via "php woorelease.phar build repo_URL" (Composer version 2.5.5, npm version 8.19.4, node version 16.20.0)

Status- Working expected with Plugin Archive/Zip file same as fix specific branch.

**Testing Environment** 

<Details>

- WordPress: 6.5.4
- Theme: Storefront 4.6.0
- Theme: Twenty Twenty-Four 1.1
- WooCommerce - 8.9.3
- PHP: 8.0.30
- Web Server: Nginx 1.20.2
- Browser: Chrome 
- OS: macOS Ventura 13.3 
- Git Branch: testing

</Details>


Next Step- Ready to Merge 🚀

**Pull request merged**